code/multitab-validation/SingleTabPage.php

```diff
@@ -16,6 +16,15 @@ class SingleTabPage extends Page
 {
     private static $table_name = 'SingleTabPage';
 
+    public function getCMSFields()
```

Comment:
`CWP\CWP\PageTypes\BasePage::getCMSFields()` adds "Comments", "Tags", "Related Pages" tabs. It means `SingleTabPage` has than a one tab.